p/themes/base-theme/template.rtl.css

```diff
@@ -719,7 +719,7 @@ input[type="search"] {
 }
 
 .flux .item.title .author {
-	padding-right: 1rem;
+	padding-left: 1rem;
```

Comment:
@Frenzie I am guessing it was a bug (there was no marking in RTL mode)

Reply:
Possibly (can't check atm), but then it's better to do it with a control directive like `/*rtl:ignore*/` in the original so the fix won't be overwritten if you run `make rtl`.